packages/third-parties/README.md

```diff
@@ -0,0 +1,51 @@
+# `@next/third-parties`
+
+`@next/third-parties` is collection of components that can be used to efficiently load third party libraries into your Next.js application.
+This package is the Next.js implementation of the [Third Party Capital](https://github.com/GoogleChromeLabs/third-party-capital/) loading recommendations.
+
+# Usage
+
+## Google Analytics
+
+The `GoogleAnalytics` component loads a Google Analytics 4 tag into the page and off-loaded the execution to a web worker using Partytown. This includes the 83 kB gtag script that is required.
```

Comment:
```suggestion
The `GoogleAnalytics` component loads a Google Analytics 4 tag into the page and off-loads the execution to a web worker using Partytown. This includes the 83 kB gtag script that is required.
```

Also I think we'll have to mention that `nextScriptWorkers` will have to be enabled for this to work (otherwise it'll fallback to loading after the page is interactive) and that it's not yet supported in the App directory: https://nextjs.org/docs/pages/building-your-application/optimizing/scripts#offloading-scripts-to-a-web-worker-experimental